Frontend.Tests/HelpersTests/TagHelperTests/GdsKeyValueTagHelperTests.cs

```diff
@@ -44,7 +44,7 @@ public void GivenKeyValues_RenderSummaryListRow()
             tagHelper.Process(_tagHelperContext, _tagHelperOutput);
 
             var expectedContent =
-                "<dt class=\"govuk-summary-list__key\">My Key</dt><dd class=\"govuk-summary-list__value dfe-summary-list__value--width-50\">My Value</dd>";
+                "<dt class=\"govuk-summary-list__key\">My Key</dt><dd class=\"dfe-summary-list__value--width-50 govuk-summary-list__value\">My Value</dd>";
```

Comment:
This is _very_ brittle - it failed locally just because the classes were in a different order in the markup.